locksmith/src/models/userReference.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+module.exports = (sequelize, DataTypes) => {
+  const UserReference = sequelize.define(
+    'UserReference',
+    {
+      emailAddress: DataTypes.STRING,
+    },
+    {}
+  )
+  UserReference.associate = function(models) {
+    UserReference.belongsTo(models.User, {
+      foreignKey: 'publicKey',
+      targetKey: 'publicKey',
+    })
+  }
+  sequelize.sync()
```

Comment:
What is this for?

Reply:
Would you want to add a `UserReference.removeAttribute('id')`?

Reply:
this is all part of the sequelize black magic, https://sequelize.readthedocs.io/en/latest/docs/associations/ . From what I understand it glues the models together and provides some of the dynamic methods for convenience